mongodbatlas/resource_mongodbatlas_maintenance_window.go

```diff
@@ -166,6 +179,13 @@ func resourceMongoDBAtlasMaintenanceWindowUpdate(ctx context.Context, d *schema.
 		}
 	}
 
+	if d.HasChange("auto_defer") {
+		_, err := conn.MaintenanceWindows.AutoDefer(context.Background(), d.Id())
```

Comment:
[nit] Same suggestion as previous one
```suggestion
		_, err := conn.MaintenanceWindows.AutoDefer(ctx, d.Id())
```